kimchi/src/circuits/lookup/constraints.rs

```diff
@@ -20,12 +21,6 @@ use CurrOrNext::{Curr, Next};
 
 use super::runtime_tables;
 
-/// Number of constraints produced by the argument.
-pub const CONSTRAINTS: u32 = 7;
```

Comment:
this shouldn't move as this is the number of constraints used by the lookup argument